domain_scoring/domain_scoring.py

```diff
@@ -15,6 +16,7 @@ def __init__(self):
         # The token_pattern also allows single character strings which the default doesn't allow
         self.vectorizer = TfidfVectorizer(analyzer='word', token_pattern='\\b\\w+\\b')
```

Comment:
We should add `tokenizer=lambda sequence: sequence.split(';')` here as mentioned by @feeds https://github.com/KDD-OpenSource/32de-python/issues/5

Reply:
Currently, all tests are green and the meta-paths are joined with a whitespace ` `. So this would only break the branch.